src/Game.elm

```diff
@@ -514,35 +464,71 @@ viewShootingStrength { state, dimensions } =
             Html.text ""
 
 
-currentPlayer : Model -> Player
-currentPlayer model =
-    case model.state of
-        PlacingBehindHeadString _ pool ->
-            EightBall.currentPlayer pool
+currentCursor : State -> String
+currentCursor state =
+    case state of
+        PlacingBallInHand HoveringOuside _ ->
```

Comment:
I know this wasn't changed with this branch, but I just noticed a typo here `HoveringOuside` is missing the "t": `HoveringOutside`.